vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -3006,11 +3009,50 @@ private void updateRoomHeaderTopic() {
                 mActionBarHeaderRoomTopic.setVisibility(View.GONE);
             } else {
                 mActionBarHeaderRoomTopic.setVisibility(View.VISIBLE);
-                mActionBarHeaderRoomTopic.setText(value);
+
+                SpannableStringBuilder strBuilder = new SpannableStringBuilder(value);
+                MatrixURLSpan.refreshMatrixSpans(strBuilder, mVectorMessageListFragment);
+                mActionBarHeaderRoomTopic.setText(strBuilder);
+
+                URLSpan[] urls = strBuilder.getSpans(0, value.length(), URLSpan.class);
+
+                if ((null != urls) && (urls.length > 0)) {
+                    for (URLSpan span : urls) {
+                        makeLinkClickable(strBuilder, span, value);
+                    }
+                }
             }
         }
     }
 
+    /**
+     * Trap the clicked URL.
+     *
+     * @param strBuilder    the input string
+     * @param span          the URL
+     */
+    public void makeLinkClickable(SpannableStringBuilder strBuilder, final URLSpan span, final String roomAlias) {
```

Comment:
`roomAlias` can be replaced by a more generic term like ... `value` (better naming is welcome).
The idea is room alias is one possible value but it can be also a room id, a user id, a group id, etc...

Reply:
I agree for use generic term value instead of roomAlias because value can be a roomId or else.